_config.yml

```diff
@@ -36,6 +36,8 @@ navigation:
     url: /blog
   - title: Docs
     url: /docs
+  - title: Benchmarks
+    url: /benchmarks
```

Comment:
I think we should hold off on adding this to the main menu.

The main menu should have something like "dashboards", then under it a drop-down where we can list benchmarks, cpython compatibility, what is left (and maybe others).

Reply:
I agree with this, I will edit it. But before I do, I have a question about this:
To what should the "CPython compatibility" and "What's left" options link to?

Reply:
@mireille-raad I have moved the link to the benchmarks page to the dashboard for now and removed it from the navbar. Let me know if this is good enough or whether you still want to have a dropdown menu added to the navbar for the dasboard.